src/KMonad/Keyboard/IO/Mac/Types.hs

```diff
@@ -214,7 +214,7 @@ kcMap = M.fromList
   , ((0x7,0x61), KeyKp9)
   , ((0x7,0x62), KeyKp0)
   , ((0x7,0x63), KeyKpDot)
-  -- , ((0x7,0x64), KeyNonUSBackslash)
+  -- , ((0x7,0x64), Key102nd)
```

Comment:
This is commented out; for a good reason I presume. Is this even a valid keycode on Mac?

Reply:
I assume it's commented out because it's designed for the ANSI keyboard that comes on macs sold in the US. But macs sold in Europe (and some other places) have an ISO layout keyboard, which has one additional key. 

The additional key is mac key code (7,100). I have a macbook pro with an ISO keyboard. When I pressed it, monad would crash. 

In order to use this key, I needed to uncomment this line, but also had to change it from KeyNonUSBackslash to Key102d. This pull request would remove that second step for anyone else with such a keyboard.

Reply:
Ah, I see; thanks for explaining!

> I don't know whether perhaps it would cause some problem for folks with an ANSI keyboard to have this uncommented (if not then uncommenting would be great!)

I… really hope not :) This would mean we're doing something evil somewhere, which would be a bug. Feel free to uncomment the line and then we can merge this

Reply:
Right, I can't see how it would cause a problem to have extra un-triggerable keycommands in this list. I certainly hope that's the case.

I uncommented it.